airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/DestinationConsumer.java

```diff
@@ -38,18 +39,19 @@
  * <li>1. Instantiate consumer.</li>
  * <li>2. start() to initialize any resources that need to be created BEFORE the consumer consumes
  * any messages.</li>
- * <li>3. Consumes ALL records via {@link DestinationConsumer#accept(T)}</li>
+ * <li>3. Consumes ALL records via {@link DestinationConsumer#accept(AirbyteMessage)}</li>
  * <li>4. Always (on success or failure) finalize by calling
  * {@link DestinationConsumer#close()}</li>
  *
- * We encourage implementing this interface using the {@link FailureTrackingConsumer} class.
+ * We encourage implementing this interface using the {@link FailureTrackingDestinationConsumer}
+ * class.
  */
-public interface DestinationConsumer<T> extends CheckedConsumer<T, Exception>, AutoCloseable {
+public interface DestinationConsumer extends CheckedConsumer<AirbyteMessage, Exception>, AutoCloseable {
```

Comment:
feels more like a `MessageConsumer` than a `DestinationConsumer`

Reply:
Nice catch. Renamed to `AirbyteMessageConsumer`.